src/components/geo/render.js

```diff
@@ -121,9 +121,11 @@ export const renderMap = ({ref, width, height, resultsData, modelViewSelected, s
 
   /* render the (text) titles */
   svg.append("text")
+    .attr("font-family", "Lato, Helvetica Neue, Helvetica, sans-serif")
```

Comment:
👍  If we want to access the `theme` object we can wrap the `<Geo>` component with `withTheme` and then hand the `theme` prop to this function ([docs here](https://www.styled-components.com/docs/advanced#getting-the-theme-without-styled-components)). This might be a good idea to keep the different D3 functions looking similar.